rasa/core/events/__init__.py

```diff
@@ -267,6 +276,8 @@ def _from_story_string(cls, parameters: Dict[Text, Any]) -> Optional[List[Event]
                     parameters.get("parse_data"),
                     parameters.get("timestamp"),
                     parameters.get("input_channel"),
+                    parameters.get("messge_id"),
```

Comment:
```suggestion
                    parameters.get("message_id"),
```

Maybe also add a test for that